.github/CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ Ensure you have `docker` and `pipenv` installed locally, and the `docker` daemon
 your local system:
```

Comment:
Don't forget to repoint the submodule to `main`

Reply:
I am suggesting we purposefully don't do this so that this merge succeeds, and then after CI/CD completes we repoint it back to `main`.

Reply:
I have a task for next sprint to fix this - https://www.wrike.com/open.htm?id=1167647370

It is blocked on another task - https://www.wrike.com/open.htm?id=1113530084 - to update the pipelines to build and push multiplatform images